build/ubuntu/Dockerfile.prebuilder

```diff
@@ -57,15 +57,17 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
         libncurses-dev \
         libncurses5 \
         libtinfo5 \
-        protobuf-compiler
+        protobuf-compiler \
+        rsync
```

Comment:
Does cabal depend on rsync? I think it is ok to add it, just asking for curiousity.

Reply:
No. Our `hack/bin/cabal-install-artefacts.sh` depends on rsync